<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelWidget</class>
 <widget class="QWidget" name="PanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1102</width>
    <height>409</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>224</red>
        <green>220</green>
        <blue>212</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>237</green>
        <blue>233</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>110</green>
        <blue>106</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>149</red>
        <green>147</green>
        <blue>141</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>199</red>
        <green>196</green>
        <blue>189</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>237</green>
        <blue>233</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>224</red>
        <green>220</green>
        <blue>212</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>237</green>
        <blue>233</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>110</green>
        <blue>106</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>149</red>
        <green>147</green>
        <blue>141</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>199</red>
        <green>196</green>
        <blue>189</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>237</green>
        <blue>233</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>110</green>
        <blue>106</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>224</red>
        <green>220</green>
        <blue>212</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>239</red>
        <green>237</green>
        <blue>233</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>110</green>
        <blue>106</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>149</red>
        <green>147</green>
        <blue>141</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>110</green>
        <blue>106</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>112</red>
        <green>110</green>
        <blue>106</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>199</red>
        <green>196</green>
        <blue>189</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>199</red>
        <green>196</green>
        <blue>189</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>224</red>
        <green>220</green>
        <blue>212</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="font">
   <font>
    <family>Sans Serif</family>
    <pointsize>8</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Панель индикации</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#mainFrame {
	border: 1px solid #303030;
    border-radius: 20px;
	background: #707070;
    padding: 2px;
}</string>
  </property>
  <widget class="QFrame" name="mainFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>1081</width>
     <height>391</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>112</green>
         <blue>112</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>112</green>
         <blue>112</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>112</green>
         <blue>112</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>112</green>
         <blue>112</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>112</green>
         <blue>112</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>112</green>
         <blue>112</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>112</green>
         <blue>112</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>112</green>
         <blue>112</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>112</red>
         <green>112</green>
         <blue>112</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>8</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="ControlRegisterPanel" name="controlRegisterPanel" native="true">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>1011</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="AdderRegisterPanel" name="adderRegisterPanel" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>1041</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="PCWidget" name="programCounterPanel" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>220</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="ArithmUnitRegisterPanel" name="arithmUnitRegisterPanel" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>1021</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="CommandRegisterPanel" name="commandRegisterPanel" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>220</y>
      <width>461</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="CommandDecoderWidget" name="commandDecoderPanel" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="StatusRegisterPanel" name="statusRegisterPanel" native="true">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="AnalogClock" name="clock" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>260</y>
      <width>121</width>
      <height>121</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>64</width>
      <height>64</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>130</width>
      <height>130</height>
     </size>
    </property>
   </widget>
   <widget class="ModesPanel" name="modesPanel" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>300</y>
      <width>481</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="StopReasonsPanel" name="stopReasonsPanel" native="true">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>300</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="DebugPanel" name="debugPanel" native="true">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>300</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>40</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ControlRegisterPanel</class>
   <extends>QWidget</extends>
   <header location="global">controlregisterpanel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AdderRegisterPanel</class>
   <extends>QWidget</extends>
   <header location="global">controlregisterpanel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PCWidget</class>
   <extends>QWidget</extends>
   <header location="global">controlregisterpanel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ArithmUnitRegisterPanel</class>
   <extends>QWidget</extends>
   <header location="global">controlregisterpanel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CommandRegisterPanel</class>
   <extends>QWidget</extends>
   <header location="global">controlregisterpanel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CommandDecoderWidget</class>
   <extends>QWidget</extends>
   <header location="global">controlregisterpanel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StatusRegisterPanel</class>
   <extends>QWidget</extends>
   <header location="global">controlregisterpanel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>AnalogClock</class>
   <extends>QWidget</extends>
   <header location="global">analogclock.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ModesPanel</class>
   <extends>QWidget</extends>
   <header location="global">controlregisterpanel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>StopReasonsPanel</class>
   <extends>QWidget</extends>
   <header location="global">controlregisterpanel.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DebugPanel</class>
   <extends>QWidget</extends>
   <header location="global">controlregisterpanel.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
